lib/templates/partials/navigation.hbs

```diff
@@ -1,19 +1,21 @@
 <div class="nav">
-    <a href="/" class="nav__logo-wrapper">
+    <a href="/" class="nav__logo__wrapper">
         <img class="nav__logo" src="/public/img/nav-logo.svg" alt="hapi pal" />
     </a>
-    <div class="nav__item-wrapper">
-        <div>
-            <a class="nav__item" href="/">
+    <div class="nav__item__wrapper">
+        <div class="nav__item__wrapper--flex">
```

Comment:
While I'm here, I don't think you need two `div`s here—you can just have `<div class="nav__wrapper nav__wrapper--flex">...`, and unless you plan on having a non-flex modifier, I think it would be totally acceptable to nix `--flex` altogether and just give `.nav__wrapper` those styles.

Reply:
another great point! fixed in the home branch!